lexibank_allenbai.py

```diff
@@ -42,21 +43,13 @@ def cmd_download(self, **kw):
 
     def cmd_install(self, **kw):
         wl = lingpy.Wordlist(self.raw.posix('Bai-Dialect-Survey.tsv'))
-        gcode = {x['NAME']: x['GLOTTOCODE'] for x in self.languages}
 
         with self.cldf as ds:
-            ds.add_sources(*self.raw.read_bib())
+            ds.add_concepts(id_factory=lambda c: c['CONCEPTICON_ID'])
+            ds.add_languages(id_factory=lambda l: l['Name'])
```

Comment:
So far, it seemed as if the only argument `add_languages|concepts` needs is a callable that returns what will become the `ID` column in `cldf/concepts|languages.csv`.